loadtesting/k6/tests/utils/accountManager.js

```diff
@@ -13,7 +13,24 @@ export function createAccount() {
     };
 }
 
-export function fundAccountWithNeon(address, amount) {
+export function fundAccountFromFaucet(client, address, amount) {
```

Comment:
so if you moved accounts preparing to clickfile do you still need these 3 functions here?

Reply:
removed